wemake_python_styleguide/violations/refactoring.py

```diff
@@ -593,7 +593,20 @@ class ImplicitElifViolation(TokenizeViolation):
 @final
 class PointlessStarredViolation(ASTViolation):
     """
+    Forbids to have useless starred expressions.
 
+    Reasoning:
+       This kind of calls is just pointless.
```

Comment:
Using starred expression with constants is useless. This piece of code can be rewritten to be flat. Eg.: `print(*[1, 2, 3])` is `print(1, 2, 3)`